src/App.js

```diff
@@ -434,10 +426,16 @@ export default class App extends Base {
           }
 
           // Set the pubSub URL if present
+          console.log('2507 test log - query params', query.params);
```

Comment:
Please remove this before merging.

Reply:
yes, will be removed once we test on sky/foxtel